docs/balloting-tool-documentation.md

```diff
@@ -0,0 +1,142 @@
+
+3. Select the repository for which giscus should be enabled and request access.
+<img src="giscus-repository-access.png" alt="giscus repository access" width="400" >
+
+5. Once access is granted go to this [website](https://giscus.app) and enter the repository name. If steps 1 and two were successful a green checkmark will indicate that giscus is enabled an can be used.
+   
+![giscus app](giscus-check.png)
+
+7. Set the discussion Category to Comment and then copy the following lines:
+
+        data-repo="HL7Austria/ELGA-FHIR-e-Medikation-R5"
+        data-repo-id="R_kgDOLhyFQQ"
+        data-category="Comment"
+        data-category-id="DIC_kwDOLhyFQc4Cev5-"
+        
+8. These values need to be updated in custom-template/content/assets/js/comment-box.js during the IG setup.
+
+
+## IG Setup
+TBD once established
+
+Ensure that the file UpdateBallotComments.yml exist in ```.github/workflows```
+
+Ensure that this line is included towards the end of fragment-pageend.html: ```<script type="text/javascript" src="assets/js/comment-box.js"></script>```
+
+
+## Final IG Ballot Checklist
+- [ ] GitHub Discussions enabled
+- [ ] "Comment" category exists
+- [ ] Giscus app installed & configured
+- [ ] `comment-box.js` updated with correct repo and repo ID
+- [ ] `UpdateBallotComments.yml` workflow exists
+- [ ] IG page includes `<script src=".../comment-box.js">`
+
+
+
+# Balloting Tool Highlighting Mechanism
+
+Upon creation of a new discussion (by pressing comment in the giscus window) a new discussion is created by the giscus bot. The creation of a new discussion triggers the UpdateBallotComments github action that is defined in ```.github/workflows/UpdateBallotComments.yml``` . 
+
+### discussion id
+
+The balloting tool relies on a unique string — referred to as the discussion ID, giscus term or highlight ID — to link each comment to a specific section of the IG. This string is used by:
+
+Giscus to fetch and identify the correct discussion thread.
+The highlighting mechanism to locate and mark the correct passage in the rendered IG.
+The term has the following structure:
+
+```[page path] | [compressed XPath] | [startOffset] | [endOffset]```
+
+Each component is separated by a pipe |. For example:
+
+```/r5-ELGA-e-medikation-section-specific-ballot-tool-approach/index.html|/html[1]/body[1]/div[4]/div[1]*4/p[3]/text()|33|85```
+
+To reduce the overall length (and avoid giscus bugs), the XPath portion is compressed using a custom scheme that replaces repeated elements (e.g., /div[1]/div[1]/div[1] becomes /div[1]*3). This compression is essential because:
+
+Giscus imposes a strict character limit on discussion titles. If this limit is exceeded, giscus silently fails to locate or render the associated discussion, without showing any error message. The compression format is reversible and the GitHub Action described below automatically decompresses it when building the comments.json used by the frontend.
+
+### UpdateBallotComments Github Action
```

Comment:
Would it be possible to determine the branch the comment was created on? Maybe as part of the URL?